### DISCUSSION TITLE
Deployment shows wrong branch information

### DISCUSSION THREAD

**Discussion by cemo** [community/community#16172] [Other Feature Feedback, Questions, & Ideas]
We use feature branch based deployments as [github flow](https://user-images.githubusercontent.com/6351798/48032310-63842400-e114-11e8-8db0-06dc0504dcb5.png).

Github deployments show information regarding `main` branch of it. 

<img width="1351" alt="image" src="https://user-images.githubusercontent.com/44156/167229059-c2b28873-7216-4372-9b52-a032ac8cf3aa.png">

However our deployments are triggered `github comments`. I see there is ambiguity regarding where it is originating but we can configure it.

I think this behavior should be customizable.

**Comment by Cazador40** [comment]
👋
